output/secondary_file.md

```diff
@@ -99,6 +99,16 @@ This makes parallel file processing easy. But if you want to disable this behavi
 dump.bin
 ```
 
+### `compress`
+
+| type | default | version |
+| :--- | :--- | :--- |
+| enum | text | 1.0.0 |
```

Comment:
Actual version which adds this feature seems 0.14.5 but it's meaning less to specify exact version.
* https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v0145---20160906
* https://github.com/fluent/fluentd/pull/1154